### ISSUE TITLE
Hyperlinks broken in KDE plasma 5.5.5

### ISSUE THREAD

**Issue by tetractius** [morkro/papyrus#13]
Even if default browser is configured correctly (and it works everywhere) I keep on getting about:black when I try opening a link.

Seems like the event is passing always _about:black_ for every link...
![image](https://user-images.githubusercontent.com/2277084/31433076-23e8c31e-ae70-11e7-984f-2014ffaad453.png)

Is there anything that I can try?
also ```x-www-browser``` seems to be configured correctly in the path and works from command line.

**Comment by morkro** [comment]
Yeah that issue actually occurs on all systems. I think it's because they use React and some tracking on the frontend and somehow the links aren't prefilled but during `onClick`.

Unfortunately, I couldn't find a solution back when I was trying to fix it. It is really annoying myself, and I will tackle this problem again soon. Any help here is very much appreciated though 🙂
